README.md

```diff
@@ -33,3 +33,83 @@ Your pull requests are welcome but please keep them manageable and focussed.
 Equally your input on any pull requests we have in flight at any given time is invaluable to us, so please get involved.
 
 Thanks! :grin:
+
+
+## Running the example
+
+* Clone the repo
+* cd to `example` folder
+* run `flutter packages get` to install dependencies
+* `flutter run` will start the application on connected android / iOS device
+
+
+## Using the Realtime API
+
+##### Import the package
+```dart
+import 'package:ably_flutter_plugin/ably.dart' as ably;
+```
+
+##### Create an Ably instance
+```dart
+final ably.Ably ablyPlugin = ably.Ably();
+```
+
+##### create a ClientOptions
+```dart
+final clientOptions = ably.ClientOptions.fromKey("<YOUR APP KEY>");
+clientOptions.environment = 'sandbox';
+clientOptions.logLevel = ably.LogLevel.verbose;
+```
+
+##### Rest API
+
+```dart
+ably.Rest rest = await ablyPlugin.createRest(options: clientOptions);
+```
+
+Getting a channel instance
+```dart
```

Comment:
A line of vertical whitespace above blocks of example code is a good idea too, please.

Reply:
Done@https://github.com/ably/ably-flutter/commit/083a0a518b96c8e773f98c8353af420b698beaa3